react/tic-tac-toe/src/app/screens/Players/index.js

```diff
@@ -1,13 +1,100 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, Component } from 'react';
+import { connect } from 'react-redux';
+import { func, arrayOf } from 'prop-types';
 
+import actionsCreators from '../../../redux/match/actions';
 import TopBar from '../Game/components/TopBar';
+import { matchPropType } from '../../../constants/propTypes';
+import { PLAYER_ONE, PLAYER_TWO, TIE } from '../../../constants/gameConstants';
+import { history } from '../../../redux/store';
 
-function Players() {
-  return (
-    <Fragment>
-      <TopBar />
-    </Fragment>
-  );
+import styles from './styles.module.scss';
+
+
+class Players extends Component {
+  componentDidMount() {
+    const { matchHistory, getMatches } = this.props;
+    if (matchHistory.length === 0) {
+      getMatches();
+    }
+  }
+
+  getMatchResult(match) {
+    if (match.winner === TIE) {
+      return { winner: false, loser: false };
+    } else if (match.winner === PLAYER_ONE) {
+      return { winner: PLAYER_ONE, loser: PLAYER_TWO };
+    }
+    return { winner: PLAYER_TWO, loser: PLAYER_ONE };
+  }
+
+  getPlayersMatches() {
+    const players = {
```

Comment:
As an improvement, I'd move this to a constant. Something like `DEFAULT_PLAYER_STATES`